### ISSUE TITLE
Detailed status view inadvertently omits jobs without eligible operation

### ISSUE THREAD

**Issue by tcmoore3** [glotzerlab/signac-flow#127]
### Description

I'm trying to use the command line interface to view the status of all of my jobs, not just ones with eligible operations. When I add the `--all-operations` option, it's showing me all operations, not just eligibile or active ones (as the documentation says it should). But I don't want to see that, I just want to see all of my jobs and their eligible operations (which would be blank if all of my post conditions are satisfied). I basically want the option to get the same behavior as before the jinja templating was used to print the status. I'm not sure if this is a missing feature from merging in #111, or missing documentation, or something else, but this should definitely be possible.

### System configuration
I'm using `master` (at 6e83bc) on stampede2:

```
$ python -c 'import platform; print(platform.platform()); import sys; print(sys.version); import signac; print(signac.__version__); import flow; print(flow.__version__)'
Linux-3.10.0-957.5.1.el7.x86_64-x86_64-with-centos-7.6.1810-Core
3.6.3 (default, Dec 11 2017, 21:18:04) 
[GCC Intel(R) C++ gcc 5.4 mode]
1.0.0
0.7.1
```

**Comment by csadorf** [comment]
@tcmoore3 Thank you for reporting. This is definitely a bug that was introduced with #111 .

**Comment by tcmoore3** [comment]
@csadorf OK, that was my gut feeling. I'm happy to fix this, but I want to confirm something: is it the intended behavior to by default only show jobs with eligible operations, which changes the behavior from the current version?

**Comment by csadorf** [comment]
The intended default behavior is to have one line per job regardless of whether there is an eligible operation pending or not. There is actually a `status` option `--only-incomplete-operations` that should switch to the behavior you described above.

**Comment by csadorf** [comment]
Resolved with merge of #128 .
